### PR TITLE
Add Appveyor integration to test all branches on windows (Closes #89)

### LINKED ISSUE #89: Create testing rig for windows
Include Windows in automated testing

### PR DESCRIPTION
**Pull request by jperata**
- Add appveyor integration
- Replaces spawn with spawn-cross to work on windows
- Make corrections on bespoke-client-test (right now have a test still failing on node 7, but it's a general issue not a windows one).
- Remove a node 4/5 issue, assign to array is invalid on that versions

Closes #89

### CODE REVIEW

File: test/client/bespoke-client-test.ts [hunk 1]
```diff
@@ -37,11 +37,21 @@ describe("BespokeClient", function() {
...
+                "proxy.bespoken.tools",
+                5000,
+                "127.0.0.1",
+                testPort);
+
             client.onConnect = function (error: any) {
                 const webhookCaller = new HTTPClient();
-                webhookCaller.post("proxy.bespoken.tools", 443, "/test?node-id=JPK" + nodeMajorVersion, "Test", function (data: Buffer, statusCode: number, success: boolean) {
-                    assert.equal(data.toString(), "BST Proxy - Local Forwarding Error\nconnect ECONNREFUSED 0.0.0.0:" + testPort);
+                webhookCaller.post("proxy.bespoken.tools",
```
Comment (chris-ramon) [inline test/client/bespoke-client-test.ts]: are those actually blank lines? :smiley:

File: lib/client/lambda-server.ts [hunk 2]
```diff
@@ -103,7 +103,9 @@ export class LambdaServer {
                 context.fail(Error(`LambdaServer input url should not contain more than '.' or node_modules.  found: ${onlyUrl}`));
                 return;
             }
-            [path, handlerFunction] = onlyUrl.split(".");
+            const splitUrl = onlyUrl.split(".");
```
Comment (chris-ramon) [inline lib/client/lambda-server.ts]: I think would be better to do array destructuring:

```typescript
const [path, handlerFunction] = onlyUrl.split(".");
```

Comment (jperata) [reply lib/client/lambda-server.ts]: that seems to be broken builds for node 4 and node 5

File: test/client/bespoke-client-test.ts (hunk 1 above)
Comment (jperata) [reply test/client/bespoke-client-test.ts]: Line was to big, so i added a line jump between attributes

**Pull request merged**